src/calculateConditionalProperties.js

```diff
@@ -85,10 +94,7 @@ export function calculateConditionalProperties(fieldParams, customProperties) {
       let fieldSetFields;
 
       if (fieldParams.inputType === supportedTypes.FIELDSET) {
```

Comment:
Regarding [@brennj 's comment](https://github.com/remoteoss/json-schema-form/pull/23#issuecomment-1618147880), did you find the cause? What schema did you use?

P.S. Github does not support threads in comments detached from code, so let's try to avoid comments in the main PR page. 😓

Reply:
Ah! I've never noticed that bug, in the playground. This is a old tricky one regarding Formik. Basically every time we change the demo (eg add new fields) Formik does not reinitialize the initial values and so it's unable to detect the `errors` of each one of the new fields. I'll fix that in the [PR for playground codebase](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/17915)

Reply:
oh then absolutely no worries then

i think this one is fine and good to go when we merge in the other PR

Reply:
[Done](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/17915/diffs#note_1455301247). I never noticed this while fixing the bug because I forced the playground (in the codebase) to render the right demo right away for a quicker DX. 😅